knip.ts

```diff
@@ -1,13 +1,15 @@
 import type { KnipConfig } from 'knip';
 
 const config: KnipConfig = {
-  // `attw` is used to ensure that our dists have proper typings.
-  ignoreBinaries: ['attw'],
+  ignore: ['packages/jest-expect-*/jest.config.js'],
```

Comment:
Knip isn't picking up on the Jest configs being used in our two packages because (I believe) `jest` is installed as a peer dep.